src/io/sampling.jl

```diff
@@ -504,6 +517,9 @@ set_factor_bounds(dom, :wave_stress, (0.1, 0.2))
 ```
 """
 function set_factor_bounds(dom::Domain, factor::Symbol, new_dist_params::Tuple)::Domain
+    if factor == :guided
+        return _update_decision_method(dom, new_dist_params)
+    end
```

Comment:
Spacing again :smile:

```suggestion
    if factor == :guided
        return _update_decision_method(dom, new_dist_params)
    end

```